Cinema system/Cinema system/Models/ViewModels/HallView.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace CinemaSystem.Models
+{
+    public class HallView
+    {
+        public int Id { get; set; }
+
+        [Required(ErrorMessage = "Name cannot be empty")]
+        [StringLength(50, ErrorMessage = "Name must be less than 50 characters")]
+        public string Name { get; set; }
+
+        [Required(ErrorMessage = "Hall cannot be empty")]
```

Comment:
You should add more details on why the hall is considered empty as it's not clear you're referring to a hall with no seats.